js/src/modal.js

```diff
@@ -83,7 +83,7 @@ class Modal extends BaseComponent {
 
     this._config = this._getConfig(config)
     this._dialog = SelectorEngine.findOne(SELECTOR_DIALOG, this._element)
-    this._backdrop = null
+    this._backdrop = this._initializeBackDrop()
```

Comment:
Was there a particular reason why you chose to use a method instead of just doing `new Backdrop()`?

Reply:
Only to separate different things, and less busy code for eyes. (maybe o future more lazy initialization?)